examples/misc_nodes_gpgpu.html

```diff
@@ -0,0 +1,295 @@
+						break;
+
+					case 'AntiPassthrough':
+						shaderNode = new ShaderNode( ( { index }, builder ) => sub( 255, index ) );
+						outAttribute = new THREE.BufferAttribute( new TypedArray( 256 ), 1 );
+						break;
+
+					case 'PassthroughColor':
+						shaderNode = new ShaderNode( ( { index }, builder ) => params.backend === 'WebGL' ? index : uvec4( index, index, index, uint( 255 ) ) );
+						outAttribute = new THREE.BufferAttribute( new TypedArray( 256 * 4 ), 4 );
+						break;
+
+					case 'ElementManipulation':
+						const srcAttribute = new THREE.BufferAttribute( new TypedArray( 256 ), 1 );
+						srcAttribute.array[ 0 ] = 255;
+						srcAttribute.array[ 7 ] = 192 - 7;
+						srcAttribute.array[ 8 ] = 64 - 8;
+						srcAttribute.array[ 15 ] = 255 - 15;
+						const srcBuffer = computationRenderer.createBuffer( srcAttribute );
+						shaderNode = new ShaderNode( ( { index }, builder ) => add( index, srcBuffer.getBufferElement( index ) ) );
+						outAttribute = new THREE.BufferAttribute( new TypedArray( 256 ), 1 );
+						break;
+
+					case 'Multiplication':
+						shaderNode = new ShaderNode( ( { index }, builder ) => {
+							const result = [ remainder( mul( index, 2 ), 256 ), remainder( mul( index, 3 ), 256 ), remainder( mul( index, 5 ), 256 ) ];
+							return params.backend === 'WebGL' ? color( ...result ) : uvec4( ...result, uint( 255 ) );
+						} );
+						outAttribute = new THREE.BufferAttribute( new TypedArray( 256 * 4 ), 4 );
+						break;
+
+					case 'Random':
+						shaderNode = new ShaderNode( ( { index }, builder ) => {
+
+							const r = hash.call( { num: add( mul( index, 4 ), 0 ) }, builder ); // @TODO: Tweak parameters to some that produce a more random image
+							const g = hash.call( { num: add( mul( index, 4 ), 1 ) }, builder );
+							const b = hash.call( { num: add( mul( index, 4 ), 2 ) }, builder );
+							const a = hash.call( { num: add( mul( index, 4 ), 3 ) }, builder ); // Works only in WebGPU
+
+							return mul( vec4( r, g, b, a ), vec4( 256, 256, 256, 256 ) );
+
+						} );
+						outAttribute = new THREE.BufferAttribute( new TypedArray( 1024 ** 2 * 4 ), 4 );
+						break;
+
+				}
+
+				const outBuffer = computationRenderer.createBuffer( outAttribute );
+				await computationRenderer.compute( shaderNode, outBuffer, params.reuploadOutput );
+				const { typedArray, buffer } = outBuffer;
+
+				if ( params.logBuffers && iteration === 0 ) console.log( params.backend, params.example, typedArray );
+
+				if ( params.backend === 'WebGL' ) {
+
+					if ( params.reuploadOutput === true ) {
+
+						material.map = new THREE.DataTexture( typedArray, buffer.image.width, buffer.image.height, buffer.format, buffer.type );
+						material.map.needsUpdate = true;
+
+					} else {
+
+						material.map = buffer;
+
+					}
+
+				} else {
+
+					if ( params.reuploadOutput === true ) {
+
+						const uint8TypedArray = uint32ToUint8( typedArray ); // WebGPU can only work with uint8 textures
```

Comment:
Thank you! I've only tried Uint8 and Uint32 -- will change the comment to just "does not support uint32 textures".

Reply:
> does not support uint32 texture

Maybe: `does not support uint32 texture sampler`

32 bit is compatible if you are not using interpolation/filtering, is possible use `textureLoad` instead of `textureSample`.
This feature we can add in the future.

https://www.w3.org/TR/webgpu/#plain-color-formats